cmd/notation/plugin.go

```diff
@@ -22,6 +22,11 @@ func pluginListCommand() *cobra.Command {
 		Use:     "list [flags]",
 		Aliases: []string{"ls"},
 		Short:   "List installed plugins",
+		Long: `List installed plugins
+
+Example - List installed Notation plugins:
+  notation plugin list
```

Comment:
I see we are using `notation cert ls` above, but not here. We should pick one style and apply it to every instances.

Reply:
Thanks. Changed `list` to `ls` to keep using short alias